skimage/morphology/misc.py

```diff
@@ -235,3 +243,208 @@ def remove_small_holes(ar, area_threshold=64, connectivity=1, *, out=None):
     np.logical_not(out, out=out)
 
     return out
+
+
+def remove_near_objects(
+    label_image,
+    min_distance,
+    *,
+    priority=None,
+    p_norm=2,
+    spacing=None,
+    out=None,
+):
+    """Remove objects until a minimal distance is ensured.
+
+    Iterates over all objects (pixels that aren't zero) inside an image and
```

Comment:
As above, I think this needs a more intuitive explanation. Are neighboring values that are equal considered an object?